src/mobile/multichoice/test/disabled.test.ts

```diff
@@ -0,0 +1,61 @@
+import { expect, fixture } from "@open-wc/testing";
+import { MobileMultiChoice } from "../index";
+
+describe("MobileMultiChoice", () => {
+  describe("disabled", () => {
+    const initItems = [
+      { label: "-----", value: "-----" },
+      { label: "Orange", value: "orange" },
+      { label: "Apple", value: "apple" }
+    ];
+
+    it("does not exists on element when initializing without props option", async () => {
+      const container = new MobileMultiChoice({
+        items: initItems
+      });
+      const el = await fixture(container);
+      const inputEl = el.querySelector(
+        ".kuc-mobile-multi-choice__input-form__select__input"
+      );
+      expect(inputEl).not.to.have.attr("disabled");
+    });
+
+    it("exists on element when initializing disabled value is true", async () => {
+      const container = new MobileMultiChoice({
+        items: initItems,
+        disabled: true
+      });
+      const el = await fixture(container);
+      const inputEl = el.querySelector(
+        ".kuc-mobile-multi-choice__input-form__select__input"
+      );
+      expect(inputEl).to.have.attr("disabled");
+    });
+
+    it("exists on element when changing by setter", async () => {
```

Comment:
@nmanhit 
Changing `exists on element when changing by setter` to `added into input element when changing from false to true by setter` will be more clearly.